Xamarin.Forms.Platform.Android/AppCompat/Platform.cs

```diff
@@ -165,7 +165,8 @@ void INavigation.RemovePage(Page page)
 
 		SizeRequest IPlatform.GetNativeSize(VisualElement view, double widthConstraint, double heightConstraint)
 		{
-			Performance.Start();
+			var reference = Guid.NewGuid().ToString();
```

Comment:
Now using a reference ID to distinguish multiple calls to the same method during a render.